packages/mapviewer/src/modules/menu/components/advancedTools/ImportCatalogue/ProviderList.vue

```diff
@@ -102,6 +375,42 @@ defineExpose({ goToFirst })
         .providers-list-item:hover {
             background-color: $list-item-hover-bg-color;
         }
+
+        .providers-group-header {
+            cursor: pointer;
+            font-weight: bold;
```

Comment:
might be better to use a `<strong>` tag instead of relying on CSS to bolden your text

Reply:
I use it for styling `<TextSearchMarker>`, I don't think it's possible to use `<strong>` tag to bold the text inside `<TextSearchMarker>`. The alternative would be create  a new prop to enable this `<strong>` tag. Is that what do you suggest?